### ISSUE TITLE
Accessibility: No name is defined for button

### ISSUE THREAD

**Issue by formulahendry** [microsoft/vscode#212321]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.89.0
- OS Version: macos sonoma version:14.4.1

Copied from https://github.com/microsoft/vscode-azureapicenter/issues/106

### GitHub Tags
#A11yTCS; #A11ySev2; #Mac; #WCAG4.1.2; #GH_VSCodeextensionforAPICenter_Mac_Apr2024; #DesktopApp; #VS Code Extension for API Center; #Name Role Value; #AILimited; #A11yMAS; #device;

### Environment Details:
Application Name: VS Code Extension for API Center
macos sonoma version:14.4.1

### Repro Steps:

1. Create a API center in azure portal.
2. Open Visual studio Code.
3. Tab till Chat option in the left side panel and press enter.
4. Type @workspace /tests in the copilot chat box.
5. Verify whether the name is defined for "How can I write unit tests for python function" button or not.

### Actual Result:
No name is defined for "How can I write unit tests for python function" button.

### Expected Result:
Name should be defined for "How can I write unit tests for python function" button.

### User Impact:
Screen reader users will not get the information about the functionality of the button if name is not defined.

### Attachment:
<img width="1032" alt="Bug_106" src="https://github.com/microsoft/vscode-azureapicenter/assets/124244478/fe9f7dcb-cae2-4ba0-85b0-ebfe3af89efc">
